code/modules/mob/living/carbon/human/life.dm

```diff
@@ -548,13 +541,13 @@
 				cells.icon_state = "charge-empty"
 
 		if(pressure)
-			pressure.icon_state = "pressure[pressure_alert]"
+			pressure.icon_state = "pressure[GET_HUD_ALERT(src, /decl/hud_element/condition/pressure)]"
 		if(toxin)
-			toxin.icon_state = "tox[toxins_alert ? "1" : "0"]"
+			toxin.icon_state = "tox[GET_HUD_ALERT(src, /decl/hud_element/condition/toxins)]"
```

Comment:
`"tox[null]"` and `"tox[0]"` are not equivalent; if the alert has never been set this might cause issues?